llama_index/indices/service_context.py

```diff
@@ -115,12 +115,10 @@ def from_defaults(
         embed_model = embed_model or OpenAIEmbedding()
         embed_model.callback_manager = callback_manager
 
-        prompt_helper = prompt_helper or PromptHelper.from_llm_predictor(
-            llm_predictor, chunk_size_limit=chunk_size_limit
-        )
+        prompt_helper = prompt_helper or PromptHelper.from_llm_predictor(llm_predictor)
```

Comment:
why are you no longer setting the chunk size limit here?

Reply:
not clear to me why user want to set chunk size limit in the prompt helper.